### DISCUSSION TITLE
docs/traefik/

### DISCUSSION THREAD

**Discussion by giscus** [ricsanfre/pi-cluster#208] [Announcements]
# docs/traefik/

How to configure Ingress Contoller based on Traefik in our Pi Kuberentes cluster.

https://picluster.ricsanfre.com/docs/traefik/

**Comment by theggers** [comment]
Hi,
there is a small typo in traefik # Basic-auth middleware
it is not removeHeaders: true
it is only removeHeader: true

**Reply by ricsanfre** [reply]
Hi @theggers,

Thanks for pointing it out.
Typo is already solved.

Regards
Ricardo

**Comment by neu7ron2** [comment]
Hi Ricardo
Any preference in using Traefik or Nginx? which one was easier to setup?

**Reply by ricsanfre** [reply]
Hi @neu7ron2, I think both of them are easy to setup. Traefik comes with GUI and it is relatively easy to configure. Nginx can be harder to configure but since it the industry-standard with a large user base, there is a large knowledge database available for doing almost any configuration. 
I started using Traefik as ingress controller for the cluster and some of the configuration required to use Traefik's CRDs. Finally I migrated all services to Nginx, so I was able to simplify the configuration and use only standard Ingress Kubernetes resources.
